addons/l10n_lv/data/account_account_tag_data.xml

```diff
@@ -0,0 +1,1504 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+
+  <record id="account_account_tag_a" model="account.account.tag">
```

Comment:
The data for the Balance Sheet and Profit and Loss is worth keeping somewhere, since it will be useful for creating those reports on top of on the new reports engine.